rr_iarrc/launch/laplacian_line_detector.launch

```diff
@@ -1,28 +1,21 @@
 <launch>
     <node name="laplacian_line_detector" pkg="rr_iarrc" type="laplacian_line_detector" output="screen">
         <!--Minimum Area to Keep-->
-        <param name="perfect_lines_min_cut" type="int" value="350" />
+        <param name="perfect_lines_min_cut" type="int" value="50" />
```

Comment:
iirc lowering the min cut before changed the behavior a lot. Is that correct, and if so, I'm curious which change(s) now make 50 a good value for the min cut?